docker-compose.yml

```diff
@@ -24,5 +27,32 @@ services:
       - .parking-service.env
     volumes:
     - images:/cowbot-images
-    #ports:
-    #  - '3000:3000'
+
+  skill-search:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
+    env_file:
+      .skill-search.env
+    volumes:
+      - ./skill-search/data/${SQLITE_DATABASE}:/app/data/${SQLITE_DATABASE}:ro
+
+  job-refresh-skills:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
+    env_file:
+      .skill-search.env
+    entrypoint: "node ./src/jobs/refreshSkills.js"
+    volumes:
+      - ./skill-search/data/${SQLITE_DATABASE}:/app/data/${SQLITE_DATABASE}:rw
+
+  job-refresh-users:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
+    env_file:
+      .skill-search.env
+    entrypoint: "node ./src/jobs/refreshUsers.js"
```

Comment:
This should probably be
`command: "./src/jobs/refreshSkills.js"`

Reply:
nevermind; no entrypoint set.  this should be `command: "node ./src/jobs/refreshSkills.js"`